test/integration/targets/ansible_log/runme.sh

```diff
@@ -2,11 +2,16 @@
 
 set -eux
 
-ALOG=${OUTPUT_DIR}/ansilbe_log_test.log
+ALOG=${OUTPUT_DIR}/ansible_log_test.log
 
 ansible-playbook logit.yml
 [ ! -f "${ALOG}" ]
 
 ANSIBLE_LOG_PATH=${ALOG} ansible-playbook logit.yml
 [ -f "${ALOG}" ]
 grep -q 'ping' "${ALOG}"
+
+rm "${ALOG}"
+# inline grep should fail if EXEC was present
+ANSIBLE_LOG_PATH=${ALOG} ANSIBLE_LOG_VERBOSITY=3 ansible-playbook -v logit.yml | tee /dev/stderr | (! grep EXEC)
```

Comment:
This is failing under macOS 13.2: https://dev.azure.com/ansible/ansible/_build/results?buildId=103411&view=logs&j=0234790f-d968-59b7-e00c-d64f03a3ea0a&t=050bde6e-6316-5116-4a50-69d2b3fe18ab&l=1806